fish-pepper/jolokia/install.dck

```diff
@@ -5,11 +5,13 @@
   var jolokiaArtifact = jolokiaVersion.match(/^2.*$/) ? "jolokia-agent-jvm" : "jolokia-jvm";
   var mavenRepo = fp.blockOpts.mavenRepo || "http://central.maven.org/maven2";
 }}# Jolokia agent
-RUN mkdir -p /opt/jolokia \
+RUN mkdir -p /opt/jolokia/etc \
  && curl {{= mavenRepo }}/org/jolokia/{{= jolokiaArtifact }}/{{= jolokiaVersion }}/{{= jolokiaArtifact }}-{{= jolokiaVersion }}-agent.jar \
          -o /opt/jolokia/jolokia.jar
 ADD jolokia-opts {{= destPath }}
 RUN chmod 444 /opt/jolokia/jolokia.jar \
- && chmod 755 {{= destPath}}
+ && chmod 755 {{= destPath}} \ 
+ && chmod 775 /opt/jolokia/etc \
+ && chgrp root /opt/jolokia/etc
```

Comment:
Here we have the assumption that there is already a group `root`  which is not the case for all flavors of unix (but for most). Its of course true in an OpenShift context, but I still suggest to make this 'openshift' behaviour configurable via a template parameter like `fp.blockOpts.useGroupMode` which, if unset, does the old 755, and if set use this as the group to use (so that it then can be called with `root`).

The conditional syntax for this templates are a bit clumpsy, I could do this conditional if you don't mind.